i18n/english.yml

```
@@ -381,13 +383,17 @@ components:
     status:
       active: Active
       all: All
+      canPublish: Can publish
       expired: Expired
       expiring-within-20-days: Expiring within 20 days
       expiring-within-5-days: Expiring within 5 days
       feedNotInDeployment: Feed not in deployment
       feedNotPublished: Feed not published
       future: Future
       no-version: No version
+      published: Published
+      publishDisabled: Publish disabled
```

I find "disabled" confusing for this state.
```suggestion
      publishDisabled: Can't publish
```

---

Suggestion from MTC: "Publish blocked"

---

addressed in [95209b3](https://github.com/ibi-group/datatools-ui/pull/1044/commits/95209b36cb9fa3a92b643b1b0d8b6e7e64cbb5c5)